core/src/main/resources/reference.conf

```diff
@@ -85,6 +85,13 @@ system {
 
   # Default number of cache read workers
   number-of-cache-read-workers = 25
+  
+  io {
+    # Global Throttling - This is mostly useful for GCS and can be adjusted to match
+    # the quota availble on the GCS API
+    number-of-requests = 100000
+    per = 100 seconds
```

Comment:
is this mapping to the "queries per 100 seconds per user" quota? 

If so I'd rather just call it that as it's easier for people to figure out what to put here.